src/app/core/admin/admin-menu/admin-menu.component.ts

```diff
@@ -122,13 +126,27 @@ export class AdminMenuComponent implements OnInit {
   }
 
   private toPlainMenuItem(item: MenuItemForAdminUi): MenuItem {
-    delete item.uniqueId; // Remove uniqueId before saving
+    const { uniqueId, isNew, ...rest } = item as any;
```

Comment:
I find the `rest` a bit confusing. Maybe it would be easier and clearer to just build a new item to be returned with each property explicitly, like we do for "// Otherwise, return as normal MenuItem" ?